src/store/rewards/saga.ts

```diff
@@ -29,6 +31,24 @@ export function* syncFetchRewards() {
   }
 }
 
+export function* checkNewRewardsLoaded() {
```

Comment:
Does this still happen when we poll for new rewards? Seems like this is wired up only for when you finish logging in but I wasn't quite sure. Maybe I'm crazy. Didn't we set it up to poll every hour?

Reply:
oh you're right. We should add this check in the after we're fetching the rewards each time i think. I have removed the delay, and added the `yield call(checkNewRewardsLoaded)` in the `fetch` (rewards).

Reply:
thanks for spotting.